DXRMissions/DeusEx/Classes/DXRMissionsM02.uc

```diff
@@ -225,6 +225,7 @@ function PreFirstEntryMapFixes()
             anna.HomeRot = vector(anna.Rotation);
         }
         ReplaceBatteryParkSubwayTNT();
+        FailIfCorpseNotHeld("TerroristCommander");
```

Comment:
bingo stuff definitely does not belong inside DXRMissions, these files don't even run in Rando Medium mode